fix repolinter issues

choeppler opened the pull request:
[repolinter](https://github.com/todogroup/repolinter) currently reports four issues:
- missing CODE_OF_CONDUCT
- missing CHANGELOG
- missing Maintainers
- missing SECURITY

To reproduce: `repolinter --rulesetUrl https://raw.githubusercontent.com/hyperledger-labs/hyperledger-community-management-tools/master/repo_structure/repolint.json`

This PR is to fix this.

choeppler committed:
fix repolinter issues: add CODE_OF_CONDUCT

Add template from hyperledger-labs/hyperledger-community-management-tools

Signed-off-by: Chris Hoeppler <christian.hoeppler@de.bosch.com>

choeppler committed:
fix repolinter issues: add SECURITY

Add template from hyperledger-labs/hyperledger-community-management-tools

Signed-off-by: Chris Hoeppler <christian.hoeppler@de.bosch.com>

manoranjith commented on CHANGELOG.md:
```diff
@@ -0,0 +1,34 @@
+# Changelog
```
@choeppler  should we include the release notes for previous versions as well ?

choeppler replied:
Yes, I think so -- feel free to add them... ;-)

choeppler replied:
Added previous versions, starting with v0.3.0 in 94b624d2

choeppler committed:
fix repolinter issues: add MAINTAINERS

Signed-off-by: Chris Hoeppler <christian.hoeppler@de.bosch.com>

choeppler committed:
fix repolinter issues: add CHANGELOG

Add CHANGELOG with release information from GitHub,
https://github.com/hyperledger-labs/perun-node/releases.

Signed-off-by: Chris Hoeppler <christian.hoeppler@de.bosch.com>

choeppler committed:
fix project name in NOTICE file

Signed-off-by: Chris Hoeppler <christian.hoeppler@de.bosch.com>
